src/server.jl

```diff
@@ -12,10 +12,10 @@ immutable MaximaSession <: Base.AbstractPipe
     output::Pipe
     process::Base.Process
 
-    function MaximaSession()
+    function MaximaSession(;ArgString="")
```

Comment:
Fixed, went with `args::String=""`, fixing the type to avoid confusion with the possibility of handing args with an array of strings.